src/Promitor.Agents.ResourceDiscovery/Graph/ResourceDiscoveryFactory.cs

```diff
@@ -30,6 +30,8 @@ public static ResourceDiscoveryQuery UseResourceDiscoveryFor(ResourceType resour
                     return new CosmosDbDiscoveryQuery();
                 case ResourceType.DataFactory:
                     return new DataFactoryDiscoveryQuery();
+                case ResourceType.DataExplorerCluster:
```

Comment:
Nit - Can you alphabetically sort please?

Reply:
Fixed nit